pkgs/by-name/an/antu-icons/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitLab,
+  hicolor-icon-theme,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "antu-icons";
+  version = "2.0"; # https://www.opendesktop.org/p/1188266/
+
+  src = fetchFromGitLab {
+    owner = "froodo_alexis";
+    repo = "Antu-icons";
+    rev = "a12a9e559b59c8ded47531e299a5516718ef9a28";
+    hash = "sha256-CLcr+X/b0moVEBV0O/dzCDq4w5G2+KRLUBdqKm0eAKA=";
+  };
+
+  propagatedBuildInputs = [ hicolor-icon-theme ];
+
+  dontDropIconThemeCache = true;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/icons/antu-icons
+    cp -r * $out/share/icons/antu-icons
+
+    # Remove or fix broken symlinks
```

```suggestion
    # Remove broken symlinks
```
we don'f "fix" anything here